src/test/mir-opt/enum_prop.rs

```diff
@@ -0,0 +1,20 @@
+// EMIT_MIR enum_prop.main.SingleEnum.diff
+
+fn main() {
+  let v = match Some(Box::new(10)) {
+    Some(x) => {
+      println!("{}", x);
```

Comment:
Try putting the `println!` and such statements into a different function - the MIR is extremely hard to read this way.

Reply:
I'll just rm, was for debugging